### PR TITLE
Update impersonate app group restriction documentation

### LINKED ISSUE #622: Add option to always allow group admins to impersonate their group members
## What Needs to be Documented?

The UI for impersonate is slightly modified. The intention behind this modification is to provide an option which would give all group admins the ability to impersonate their respective members. So eventually the admin does have option to:
1. Grant access to all group admins to impersonate their members
2. Grant access to selective group admins to impersonate their members ( which we already had )

The UI would look like this:
1. 
impersoante_first
2. If the user selects the first option 
impersonate_second
all group admins in the oC instance would get access to their members.
3. If user selects the second radio button 
impersonate_third
only selective group admins would have access to their members. This is same old behaviour.

Dev PR related to this task https://github.com/owncloud/impersonate/pull/134

## Why Should This Change Be Made? (Optional)

It helps admins to grant access to all group admins, instead of providing each group admin name individually.

## What Type Of Content Change Is This? (Optional)
- [ ] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [ ] Bug Fix to Existing Content

## Which Manual Does This Relate To? (Optional)
- [ ] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

### PR DESCRIPTION
**Pull request by settermjd**
This fixes #622 by updating the text in the "Restrict Impersonation to Groups & Group administrators" section to reflect the changes to the Impersonate app in version 0.5.0.

### CODE REVIEW

File: modules/admin_manual/pages/issues/impersonate_users.adoc [hunk 1]
```diff
@@ -31,37 +32,32 @@ see.
...
+For example, if an ownCloud administrator restricts user impersonation only to the group: `group1`, then *only* `group1`'s administrators can impersonate users belonging to `group1`. 
+No other users can impersonate other users.
 
 NOTE: ownCloud administrators can always impersonate all users of an ownCloud instance 
 when the application is installed.
 
-To enable this option, in the administrator settings panel
-(`administrator -> Settings -> Admin`) in the "**User Authentication**"
-section, you’ll see a section titled: "**Impersonate Settings**"; which you can see below.
+By default, when the Impersonate app is installed, *all* group admins will be allowed to impersonate users within the groups they administer.
```
Comment (sharidas) [inline modules/admin_manual/pages/issues/impersonate_users.adoc]: By default, when impersonate app is installed, only admin user ( or say administrator) can only impersonate any users. Only if any of the radio buttons are clicked 
impersonate_settings
the behaviour changes according to the selection.

Comment (settermjd) [reply modules/admin_manual/pages/issues/impersonate_users.adoc]: Thanks for the clarification.

Comment (settermjd) [reply modules/admin_manual/pages/issues/impersonate_users.adoc]: When I installed the latest version, "_Allow group admins of specific groups to impersonate the users within those groups_" was already selected. Secondly, what do you mean by "_( or say administrator)_"?

**Comment by settermjd** [comment]
@sharidas, can you review the most recent changes, and let me know if you're happy with them?

File: modules/admin_manual/pages/issues/impersonate_users.adoc [hunk 2]
```diff
@@ -31,37 +32,36 @@ see.
...
+
+To configure it, in the administrator settings panel, which you can find under menu:administrator[Settings > Admin > User Authentication], you'll see a section titled: "**Impersonate Settings**" (which you can see below).
 
 image:apps/impersonate/impersonate-settings.png[Impersonate App settings]
 
-Check the checkbox under that, and a textbox will appear. If you click
-in the textbox, you will see a list of available groups on your ownCloud
-installation. As you type, the list will filter down to only ones that
-match the text entered, as you can see below.
+If you want to allow group admins to users within groups they administer, click btn:[Allow all group admins to impersonate users within the groups they are admins of].
```
Comment (sharidas) [inline modules/admin_manual/pages/issues/impersonate_users.adoc]: I might be wrong here. But I was wondering shouldn't this be rephrased as:
`If you want to allow all group admins to impersonate users within group they administer...`. If my thinking/observation is wrong, then let me know I will approve the PR right away. Thnaks.

Comment (settermjd) [reply modules/admin_manual/pages/issues/impersonate_users.adoc]: You're correct. I'll fix my mistake.

Comment (settermjd) [reply modules/admin_manual/pages/issues/impersonate_users.adoc]: Update made, @sharidas.

**Comment by settermjd** [comment]
Should this be backported to 10.0 as well as 10.1?

**Pull request merged**